test/unit/math/binaryOps/src/binaryOps.cpp

```diff
@@ -0,0 +1,432 @@
+                        { nullptr,           &std::max,      &alpaka::math::max<TAcc, Data, Data>,       Range::Y_NOT_ZERO},
+                        { nullptr,           &std::min,      &alpaka::math::min<TAcc, Data, Data>,       Range::UNRESTRICTED},
+                        { &std::pow,         nullptr,        &alpaka::math::pow<TAcc, Data, Data>,       Range::POSITIVE_ONLY},
+                        { &std::remainder,   nullptr,        &alpaka::math::remainder<TAcc, Data, Data>, Range::POSITIVE_ONLY}
+                };
+
+        Idx const numOps = sizeof(arr) / sizeof(TestStruct<TAcc, Data>);
+        Idx const elementsPerThread(1u);
+        Idx const sizeArgs(10u);
+        Idx const sizeRes = sizeArgs * numOps;
+
+        // Create the kernel function object.
+        BinaryOpsKernel kernel;
+
+        // Get the host device.
+        auto const devHost(
+                alpaka::pltf::getDevByIdx<PltfHost>(0u));
+
+        // Select a device to execute on.
+        auto const devAcc(
+                alpaka::pltf::getDevByIdx<PltfAcc>(0u));
+
+        // Get a queue on this device.
+        QueueAcc queue(devAcc);
+
+        alpaka::vec::Vec<Dim, Idx> const extent(numOps);
+
+        // Let alpaka calculate good block and grid sizes given our full problem extent.
+        alpaka::workdiv::WorkDivMembers<Dim, Idx> const workDiv(
+                alpaka::workdiv::getValidWorkDiv<TAcc>(
+                        devAcc,
+                        extent,
+                        elementsPerThread,
+                        false,
+                        alpaka::workdiv::GridBlockExtentSubDivRestrictions::Unrestricted));
+
+
+        // Allocate host memory buffers.
+        auto memBufHostArgsX(alpaka::mem::buf::alloc<Data, Idx>(devHost, sizeArgs));
+        auto memBufHostArgsY(alpaka::mem::buf::alloc<Data, Idx>(devHost, sizeArgs));
+        auto memBufHostRes(alpaka::mem::buf::alloc<Data, Idx>(devHost, sizeRes));
+        auto memBufHostStructs(alpaka::mem::buf::alloc<TestStruct<TAcc, Data>, Idx>(devHost, extent));
+
+        Data *const pBufHostArgsX = alpaka::mem::view::getPtrNative(memBufHostArgsX);
+        Data *const pBufHostArgsY = alpaka::mem::view::getPtrNative(memBufHostArgsY);
+        Data *const pBufHostRes = alpaka::mem::view::getPtrNative(memBufHostRes);
+        TestStruct<TAcc, Data> *const pBufHostStructs = alpaka::mem::view::getPtrNative(memBufHostStructs);
+
+        // This is just for a better understanding which results are unchanged.
+        for (Idx i(0); i < numOps; ++i)
+        {
+            for (Idx j(0); j < sizeArgs; ++j) {
+                pBufHostRes[j + i * sizeArgs] = -1;
+            }
+        }
+
+        // C++11 random generator for uniformly distributed numbers in [-100, 100].
+        std::random_device rd{};
+        std::default_random_engine eng{rd()};
+        std::uniform_real_distribution<Data> dist(0, 100);
```

Comment:
could you please define the `100` as `constexpr` in this file (maybe at the top) and use the variable name later in the code.